src/components/pages/Listings/SearchListingByName.tsx

```diff
@@ -0,0 +1,96 @@
+import { searchANT } from '@blockydevs/arns-marketplace-data';
+import {
+  Button,
+  Paragraph,
+  SearchInput,
+} from '@blockydevs/arns-marketplace-ui';
+import { useGlobalState } from '@src/state';
+import eventEmitter from '@src/utils/events';
+import { BLOCKYDEVS_ACTIVITY_PROCESS_ID } from '@src/utils/marketplace';
+import { useMutation } from '@tanstack/react-query';
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+const SearchListingByName = () => {
+  const [searchValue, setSearchValue] = useState('');
+  const [{ aoClient, arioProcessId }] = useGlobalState();
+  const navigate = useNavigate();
+  const mutationSearch = useMutation({
+    mutationFn: async (name: string) => {
+      return searchANT({
+        name,
```

Comment:
We need to encode this to ASCII + puny code for searches. Dunno if the data lib  does this internally (good idea to actually if it doesn't already). Likewise when rendering the names we should be encoding to unicode.
```suggestion
        name: lowerCaseDomain(name),
```